conans/test/functional/command/config_test.py

```diff
@@ -21,8 +22,9 @@ def storage_test(self):
         self.assertIn("path = ./data", self.client.user_io.out)
```

Comment:
Should this value be computed also? I don't care if we do it now but maybe add it to the issue if you didn't.

Reply:
Done, annotated in https://github.com/conan-io/conan/issues/5351